packages/documentation/src/stories/getting-started/migration-guide/setup.component.ts

```diff
@@ -16,13 +17,37 @@ export class SetupComponent extends LitElement {
   render() {
     return html`
       <section>
-        <div class="row migration-options mb-huge">
-          <div class="col-md-6">
+        <div class="row gap-large migration-options">
+          <div class="col-auto">
+            <label class="form-label font-curve-small bold" for="docs_Default_ExampleSelect">
+              What version of the Design System is your application currently using?
+            </label>
+            <select
+              @change="${this._onVersionChange}"
+              id="docs_Default_ExampleSelect"
+              class="form-select form-select-lg"
+            >
+              <option value="v6-to-v7" ?selected="${this.version === 'v6-to-v7'}">
+                @swisspost/design-system-styles 6.x.x
+              </option>
+              <option value="v5-to-v6" ?selected="${this.version === 'v5-to-v6'}">
+                @swisspost/design-system-styles 5.x.x
+              </option>
+              <option value="v4-to-v5" ?selected="${this.version === 'v4-to-v5'}">
+                @.../common-web-frontend 4.x.x or lower
+              </option>
+            </select>
```

Comment:
Nitpick: it looks a bit misleading. As a user I want to upgrade from v4 to v7 and when I select the v4 it tells me to upgrade to v5. I could think that I can't really upgrade to the latest. Wouldn't be easier to get all the three steps in the same page in that case?

Reply:
I just updated the version selector: now if you select v4 as your current version you will see all migration guides to migrate from 4 to 5, then from 5 to 6, and finally from 6 to 7 in that order. What do you think?

Reply:
Perfect 👌